integration-tests/terraform/eks/deployment/variables.tf

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+variable "region" {
+  type    = string
+  default = "us-west-2"
+}
+
+variable "k8s_version" {
+  type    = string
+  default = "1.24"
+}
+
+variable "test_dir" {
```

Comment:
Dont think this is being used anywhere

Reply:
This is being used in main.tf validator resource